test/deploy/kubernetes/manual/destroy.sh

```diff
@@ -1,77 +1,92 @@
 #!/bin/bash
-SCRIPT_DIR=$(dirname $0)
-MAXNODES="3"
+BASEPATH=$(dirname $0)
+MAXNODES=$(kubectl get nodes | awk -v col=1 '{print $col}' | tail -n+2 | wc -l)
 NAMESPACE="cortx"
 
 function print_header {
-    echo -e "---------------------------------------------------"
+    echo -e "--------------------------------------------------------------------------"
```

Comment:
-e option is is redundant for all the echo statements in this function except "echo -e "$1"", can be removed.